source/documentation/using_the_api/getting_updates.md

```diff
@@ -6,106 +6,38 @@ You can regularly compare this value with new queries to a register to see if th
 
-For example, your last highest entry number from the `country` register could
-be 205. You could then use the following request to find out if there are any
-new entries:
+For example, your last highest entry number from the `government-organisation` register could
+be 750. You could then make this request to find out if there are any new entries:
 
 ```http
-GET /entries/?start=206 HTTP/1.1
-Host: country.register.gov.uk
+GET /entries/?start=751 HTTP/1.1
+Host: goverment-organisation.register.gov.uk
 Accept: application/json
 Authorization: YOUR-API-KEY-HERE
 ```
 
-Example response:
+If you receive a response, you'll know there are updates.
 
-```http
-HTTP/1.1 200
-Content-Type: application/json
-Link <?start=106&limit=100>; rel="previous"
-
-[
-  {
-    "index-entry-number": "206",
-    "entry-number": "206",
-    "entry-timestamp": "2017-03-29T14:22:30Z",
-    "key": "GM",
-    "item-hash": [
-      "sha-256:0429375c4fb403288ef816e5dd38a24f192e35b8f55e40cc6266eb25eaef77b1"
-    ]
-  },
-  {
-    "index-entry-number": "207",
-    "entry-number": "207",
-    "entry-timestamp": "2017-10-25T09:52:52Z",
-    "key": "CI",
-    "item-hash": [
-      "sha-256:b3ca21b3b3a795ab9cd1d10f3d447947328406984f8a461b43d9b74b58cccfe8"
-    ]
-  },
-  {
-    "index-entry-number": "208",
-    "entry-number": "208",
-    "entry-timestamp": "2018-06-13T13:54:40Z",
-    "key": "SZ",
-    "item-hash": [
-      "sha-256:f89f36ed8b2a1417237a8e95b810e8ab4ead844277ad7bc7794cb5f83732c976"
-    ]
-  }
-]
-```
-
-### See what data is in each entry 
-
-You can use the [`GET /items/{item-hash}` endpoint](#items) on the value in
-the `item-hash` property. 
-
-Following the previous example, you could make a request using the
-`item-hash` value in entry 208:
-
-```http
-GET /items/sha-256:f89f36ed8b2a1417237a8e95b810e8ab4ead844277ad7bc7794cb5f83732c976
-Host: country.register.gov.uk
-Accept: application/json
-Authorization: YOUR-API-KEY-HERE
-```
-
-Example response:
+You should also look at the `Link` header. In the example response:
 
 ```http
 HTTP/1.1 200
 Content-Type: application/json
-{
-  "country": "SZ",
-  "official-name": "Kingdom of Eswatini",
-  "name": "Eswatini",
-  "citizen-names": "Swazi"
-}
+Link: <?start=850&limit=100>; rel="next",<?start=650&limit=100>; rel="previous"
 ```
 
-### If you have more updates than the value of `limit` 
+Here, there are more than 100 new entries since the last update, as shown by `rel="next"`. 
 
-[The `limit` parameter defaults to 100, and its maximum value is
-5000](#get-entries). In the previous example, `limit` was 100. Here, if there
-were more than 100 new entries since your last update, you would not get them
-all. 
+In this situation, you could set the `limit` parameter higher. You could also make new requests, increasing the value of `start` each time, until a `Link` header is not returned with `rel="next"`.
```

Comment:
I think this paragraph should remove any mention to the `limit` parameter. The point of using `Link` with `rel="next"` links is to not have to explain how pagination works so this should be more on the lines of:

> In this situation you have to make a new request following the link in `rel="next"`, if the next page has another `rel="next"` you know there are entries you still need to consume, the latest page will not have any `rel="next"`.